local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
```

Comment:
As stated here, please remove this file and add it to .gitignore to prevent future commits from adding it.

Reply:
This has not been resolved yet, so don't close it